lib/Controller/BanController.php

```diff
@@ -102,7 +104,16 @@ public function listBans(): DataResponse {
 	 */
 	#[PublicPage]
 	#[RequireModeratorParticipant]
-	public function unbanActor(int $banId): DataResponse {
+	public function unbanActor(): DataResponse {
```

Comment:
banId should be taken from the URL
But I just noticed I missed the parameter there:
https://github.com/nextcloud/spreed/blob/main/appinfo/routes/routesBanController.php#L20
should be:
```
['name' => 'Ban#unbanActor', 'url' => '/api/{apiVersion}/ban/{token}/{banId}', 'verb' => 'DELETE',…
```
there